.github/workflows/codecov.yml

```diff
@@ -0,0 +1,26 @@
+name: Unit Test Coverage
+
+on: [push]
+jobs:
+  coverage:
+    name: unit test coverage
+    runs-on: ubuntu-latest
+    steps:
+       - uses: actions/checkout@master
+       - name: rebuild taxonomies
+         run:  |
+          git ls-files taxonomies/ | xargs -I{} git log -1 --date=format:%Y%m%d%H%M.%S --format='touch -t %ad "{}"' "{}" | bash
```

Comment:
cool, can you add a comment on this, for our future us ;-)

Reply:
ignore it if you just copy/pasted it from the other file, in this case I might be the culprit :grimacing: